scene/gui/tree.cpp

```diff
@@ -3324,14 +3325,20 @@ void Tree::gui_input(const Ref<InputEvent> &p_event) {
 						int len = 0;
 						for (int i = 0; i < columns.size(); i++) {
 							len += get_column_width(i);
-							if (pos.x < len) {
-								emit_signal(SNAME("column_title_pressed"), i);
+							if (pos.x < static_cast<float>(len)) {
```

Comment:
`pox.x` is `real_t` (either `float` or `double`) so the cast should be to `real_t`. But I don't think casting is needed at all, it would be impilcitly converted anyway.

Reply:
Good point, I missed it was real_t.   The cast is there to clarify to the static code analyzer that this is intentional and not a mistaken narrowing.  Recent analyzer in MSVC tags this as a probable bug because of loss of precision in the implicit cast.

Will correct to the right typedef though!

Reply:
Changed via force push